### PR TITLE
Add missing handlers and templates for web based output

### PR DESCRIPTION
**Pull request by pschwartz**
Signed-off-by: Philip Marc Schwartz <philip@progmad.com>

### CODE REVIEW

File: web/endpoints/status/status.go [hunk 1]
```diff
@@ -110,6 +143,101 @@ func StatusBucketListHandleFunc(w http.ResponseWriter, r *http.Request) {
...
+	p := &Model{Date: database.DStamp, Dates: buckets}
+
+	h := templates.Lookup("header.tmpl")
+	l := templates.Lookup("list.tmpl")
+	f := templates.Lookup("footer.tmpl")
+
+	h.ExecuteTemplate(w, "header", p)
+	l.ExecuteTemplate(w, "list", p)
+	f.ExecuteTemplate(w, "footer", nil)
+	templates.Execute(w, nil)
```
Comment (rattboi) [inline web/endpoints/status/status.go]: These last 8 lines keep showing up. How about refactoring into a separate function?

**Review by rattboi — CHANGES_REQUESTED** [review]
Looks good other than my refactor suggestion

**Review by rattboi — DISMISSED** [review]
LGTM :+1:

File: web/endpoints/status/status.go [hunk 2]
```diff
@@ -3,17 +3,74 @@ package status
...
 	jww "github.com/spf13/jwalterweatherman"
 
 	"github.com/bonnyci/quartermaster/database"
 	"github.com/bonnyci/quartermaster/web/engine"
 	"github.com/bonnyci/quartermaster/web/middleware"
 )
 
+var tempPath = "web/endpoints/status/templates/"
+
+// template loading adopted from https://elithrar.github.io/article/approximating-html-template-inheritance
```
Comment (SpamapS) [inline web/endpoints/status/status.go]: The MIT license has to be included, in full, with any copies of the code. Luckily that's the only restriction, so Apache 2.0 can be applied on top of it (so the code is effectively dual-licensed). So what I'd suggest is putting the code in question in its own file with an MIT license header, or if you can't split it out, adding it right here under this comment.

**Review by SpamapS — DISMISSED** [review]

**Comment by pschwartz** [comment]
@rattboi @SpamapS Please re-review.

**Review by rattboi — APPROVED** [review]

**Pull request merged**